sdk/spring/azure-spring-boot-core/README.md

```diff
@@ -23,5 +21,4 @@ To suggest a new feature or changes that could be made, file an issue the same w
 You can participate community driven [![Gitter](https://badges.gitter.im/Microsoft/spring-on-azure.svg)](https://gitter.im/Microsoft/spring-on-azure)
 
 <!-- LINKS -->
```

Comment:
I think it is a big change, we should write some context here

Reply:
I like @backwind1233 's idea, we usually should keep the README file to do it's own job -- give reader more context aobut what is this project for ?